src/application/components/JsonDoc/Documentation.js

```diff
@@ -12,23 +13,61 @@ const defaultView =
     <span className={styles.defaultView}>&lt;-- Hover over JSON example for documentation</span>
   </div>);
 
-const Documentation = ({ documentationFullType, service, importedServices }: {
+type Props = {
   documentationFullType: string,
   service: Service,
   importedServices: Service[],
-}) => {
-  if (!documentationFullType) return defaultView;
-
-  const modelName = documentationFullType.substring(0, documentationFullType.lastIndexOf('.'));
-  const fieldName = documentationFullType.substring(documentationFullType.lastIndexOf('.') + 1);
-  const model = utils.getModel(modelName, service, importedServices);
-  const field = model ? model.fields.find(f => f.name === fieldName) : null;
-
-  return (
-    <div className={styles.documentation}>
-      <ParameterList {...field} service={service} importedServices={importedServices} parentModel={modelName} />
-    </div>
-  );
 };
 
+class Documentation extends Component {
+  props: Props;
+  state: {
+    height: number,
+  };
+
+  constructor(props: Props) {
+    super(props);
+    this.state = { height: 0 };
+    this.updateContainerHeight = this.updateContainerHeight.bind(this);
+  }
+
+  componentDidUpdate() {
+    this.updateContainerHeight();
+  }
+
+  container: HTMLDivElement;
+  updateContainerHeight: () => void;
+
+  updateContainerHeight() {
```

Comment:
Any reason not to use a fat arrow function here? Would mean you don't have to bind.